Assets/Scripts/UI/Menu's/MainMenuController.cs

```diff
@@ -0,0 +1,21 @@
+using UnityEngine.SceneManagement;
+using UnityEngine;
+
+public sealed class MainMenuController : MenuController
+{
+    [SerializeField] private string mainScene;
+    [SerializeField] private GameObject activeExplanationMenu;
+    [SerializeField] private GameObject activeMainMenu;
+
+    public void PlayGame()
+    {
+        Time.timeScale = 1;
+        SceneManager.LoadSceneAsync(mainScene);
+    }
```

Comment:
Waarvoor was ookalweer de Time.timeScale = 1 hier nodig? Het voelt nu heel random dat dit hier er is.

Als je dit er uitkan halen kan je deze functie gewoon zo neerzetten:
```suggestion
    public void PlayGame() => SceneManager.LoadSceneAsync(mainScene);
```